cypress/e2e/models/administration/assessment_questionnaire/assessment_questionnaire.ts

```diff
@@ -15,8 +23,48 @@ export class AssessmentQuestionnaire {
         });
     }
 
-    public importQuestionnaire() {
+    public static importQuestionnaire(fileName) {
         AssessmentQuestionnaire.open();
+        click(importQuestionnaire);
+        cy.get(questionnaireUpload, { timeout: 2 * SEC }).attachFile(fileName, {
+            subjectType: "drag-n-drop",
+        });
+        cy.get(controlsForm, { timeout: 5 * SEC })
+            .find("button")
+            .contains("Import")
+            .click();
+    }
+
+    public static deleteQuestionnaire(fileName) {
+        AssessmentQuestionnaire.open();
+        cy.contains(fileName, { timeout: 120 * SEC })
+            .closest("tr")
+            .within(() => {
+                click(actionButton);
+            });
+        clickByText(button, deleteAction);
+        cy.get(confirmDeletion).click().focused().clear().type(fileName);
+        clickByText(button, deleteAction);
+    }
+
+    public static exportQuestionnaire(fileName) {
+        AssessmentQuestionnaire.open();
+        cy.contains(fileName, { timeout: 120 * SEC })
+            .closest("tr")
+            .within(() => {
+                click(actionButton);
+            });
+        clickByText(button, "Export");
+    }
+
+    public static viewQuestionnaire(fileName) {
+        AssessmentQuestionnaire.open();
```

Comment:
These two lines are common to all methods and can be moved a method called "selectQuestionnaire()"

Reply:
Added  a new method called operation()